### ISSUE TITLE
jinja2 yaml pretty printer filter

### ISSUE THREAD

**Issue by ademaria** [deepmind/kapitan#27]
I would like to create a INVENTORY.md to allow for looking at the inventory without running `kapitan inventory`, so that users can understand what values are set to without running kapitan.

The INVENTORY.md doc contains the following
```
{{inventory.classes | pprint}}


{{inventory.parameters | pprint }}
```

This would look much better if there was a proper toyaml jinja filter.

**Comment by gburiola** [comment]
I believe this is a feature for jinja a not so much for kapitan.

jinja only supports `tojson` and it looks a bit better than `pprint` but there isn't a `toyml`

```
{{inventory.classes | tojson(indent=2) }}
```

Otherwise, you can use use jsonnet and kapitan `output_type: yaml`

```
  kapitan:
    compile:
    - output_path: variables
      input_type: jsonnet
      input_paths:
        - components/variables/main.jsonnet
      output_type: yaml
```

```
local kube = import "lib/kube.libjsonnet";
local kap = import "lib/kapitan.libjsonnet";
local inv = kap.inventory();

{
    "classes": inv.classes,
    "parameters": inv.parameters,
}

```

**Comment by ademaria** [comment]
I want to be able to add text features i.e. code sections and possibly links.

**Comment by adrianchifor** [comment]
Implemented in #32 . Can be used by doing:
```
{{inventory.classes | yaml}}

{{inventory.parameters | yaml}}
```
